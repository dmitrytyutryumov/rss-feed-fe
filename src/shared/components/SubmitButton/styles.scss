@import 'src/app/styles/vars.scss';

.submit-btn {
  min-height: 2.5rem;

  border-radius: 0.4rem;

  font-size: $fs-500;
  font-weight: 700;
  color: white;

  background-color: $color-primary;
}