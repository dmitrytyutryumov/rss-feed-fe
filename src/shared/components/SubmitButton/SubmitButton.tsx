import './styles.scss'

export function SubmitButton({ className, children, ...props }: any) {
  console.log(className)
  const btnStyle = 'submit-btn' + ` ${className}` || ''
  return (
    <button className={btnStyle} type="submit" {...props}>
      {children || 'Submit'}
    </button>
  )
}
