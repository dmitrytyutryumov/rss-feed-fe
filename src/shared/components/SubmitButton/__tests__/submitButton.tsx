import { render } from '@testing-library/react'
import { SubmitButton } from '../SubmitButton'

describe('test submit button', () => {
  it('test sumbit button', () => {
    // Act
    const { asFragment } = render(<SubmitButton />)
    // Assert
    expect(asFragment()).toMatchSnapshot()
  })
})
