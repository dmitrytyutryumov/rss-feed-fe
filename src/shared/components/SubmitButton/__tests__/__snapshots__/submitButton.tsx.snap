// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test submit button test sumbit button 1`] = `
<DocumentFragment>
  <button
    class="submit-btn undefined"
    type="submit"
  >
    Submit
  </button>
</DocumentFragment>
`;
